### PR TITLE
Update CRAD.Rd

### PR DESCRIPTION
**Pull request by DataMining-ClusteringAnalysis**

### CODE REVIEW

File: funtimes/man/CRAD.Rd [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+\alias{crad}
 \title{CRAD Clustering Algorithm}
 
 \description{
 A density-based clustering algorithm, named CRAD, which is based on an adaptive neighbor searching function and a robust data depth as the dissimilarity measure.
 }
 
 \usage{
-crad(X, StepSize, Nbin, Rcov = FALSE)
+crad(X, Nbin, StepSize, Rcov = FALSE)
```
Comment (vlyubchich) [inline funtimes/man/CRAD.Rd]: Still doesn't match the code. Can you just use copy-paste from code?

File: funtimes/man/CRAD.Rd [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
+\item{X}{a \eqn{n \times k} matrix where \eqn{n} is a number of observations and \eqn{k} is a number of features. }
 
-\item{Nbin}{the number of bins in generating histogram H.}
+\item{Nbin}{the number of bins in generating histogram \code{H}.}
 
-\item{StepSize}{the size of neighbor buckets in histogram H to compare for each \eqn{h_{i},i = 1, 1-width, 1-2*width, . . . , width}. Default value is 1.}
+\item{StepSize}{the size of neighbor buckets in histogram \code{H} to compare for each \eqn{h_{i},i = 1, 1-width, 1-2*width, . . . , width}. Default value is 1.}
 
-\item{Rcov}{a robust covariance matrix estimator, Minimum Covariance Determinant (MCD), will be used. If False (a default option), a sample covariance matrix \code{cov(X)} will used as the estimator.}
+\item{Rcov}{a robust covariance matrix estimator, Minimum Covariance Determinant (MCD), will be used when \code{Rcov = FALSE} (a default option), or the sample covariance matrix \code{cov(X)}.}
```
Comment (vlyubchich) [inline funtimes/man/CRAD.Rd]: Should be reverse, imho. If 'Rcov' means robust estimation, then when it is TRUE, then the robust estimator is applied.

**Review by vlyubchich — CHANGES_REQUESTED** [review]
Please, see 2 comments.

**Comment by DataMining-ClusteringAnalysis** [comment]
Hi Slava, Thanks for pointing it out. I just updated the description. What do you think this time?

**Pull request merged**